File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/satl/VerifyConfirmResponse.java
package info.nightscout.androidaps.plugins.pump.insight.satl;

import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf;

public class VerifyConfirmResponse extends SatlMessage {

    private PairingStatus pairingStatus;

    @Override
    protected void parse(ByteBuf byteBuf) {
        pairingStatus = PairingStatus.Companion.fromId(byteBuf.readUInt16LE());
    }

    public PairingStatus getPairingStatus() {
        return this.pairingStatus;
    }
}

File: insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/satl/VerifyConfirmResponse.kt
package info.nightscout.androidaps.plugins.pump.insight.satl

import info.nightscout.androidaps.plugins.pump.insight.utils.ByteBuf

class VerifyConfirmResponse : SatlMessage() {

    lateinit var pairingStatus: PairingStatus
        private set

    override fun parse(byteBuf: ByteBuf) {
        pairingStatus = PairingStatus.fromId(byteBuf.readUInt16LE())
    }
}